docs/hopr-documentation/versioned_docs/version-v1.92/node/using-docker.md

```diff
@@ -148,6 +148,27 @@ If you are in the process of registering your node on the network registry, plea
 
 Otherwise, the installation process is complete! You can proceed to our [hopr-admin tutorial](using-hopr-admin).
 
+### Breakdown of arguments
+
+```
+hoprd
+  --restart on-failure -m 2g                  # restart node if it crashes or consumes 2GB of memory 
+                                                (-m 2g)
+  --log-opt max-size                          # maximum size of a single log file, will use a new one
+                                                if reached
+  --log-opt max-file                          # maximum number of log files per docker container
+  --identity /app/hoprd-db/.hopr-identity     # store your node identity information in the
+                                                persisted database folder
+  --password switzerland   	                  # set the encryption password for your identity
+  --init 				                      # initialize the database and identity if not present
+  --announce 				                  # announce the node to other nodes in the network 
+                                                and act as a relay
+  --host "0.0.0.0:9091"   	                  # set IP and port of the P2P API to the container's
+                                                external IP
+  --apiToken <MY_TOKEN>                       # specify password for accessing REST API(REQUIRED)
```

Comment:
Not necessarily required if you use `--disableApiAuthentication` - but that is only recommend if the user does not intend to expose their API port (default 3001) to public.